plugins/quartertiler/Main.vala

```diff
@@ -15,22 +15,15 @@
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 
-using Clutter;
-using Meta;
-
 namespace Gala.Plugins.QuarterTiler
 {
 	public class Main : Gala.Plugin
 	{
 		public override void initialize (Gala.WindowManager wm)
 		{
-			Screen screen = wm.get_screen ();
-			Display display = screen.get_display ();
+			unowned Meta.Display display = wm.get_screen ().get_display ();
+
 			var settings = new GLib.Settings (Config.SCHEMA + ".keybindings");
-			if (!("tile-topleft" in settings.list_keys ())) {
-					warning ("Quarter tiling key bindings not found");
-					return;
-			}
```

Comment:
@ricotz Thanks for the cleanup! But why remove this check? Without this, if a user where to try gala with this version and then downgrade he wouldn't be able to login anymore. 
See: https://elementaryos.stackexchange.com/a/17582/3892

Reply:
gsettings schemas and keys are part of the API. We can't deal with not properly installed systems of all kinds.

Reply:
After I've experiencing suddenly not being able to login that seems scary / fragile, but yeah it probably doesn't make sense to make an exception for this property.